### DISCUSSION TITLE
Networking help

### DISCUSSION THREAD

**Discussion by Dimentagon** [apohl79/audiogridder#961] [Help] [answered]
I have two M1 macs (macPro 14" & a Mac mini) in seperate rooms and a Windows 10 intel i7 NUC. Currently I have the MacBook Pro (running the Daw plugin) connected to the Windows NUC i7 as the Server directly via Ethernet. My question is what is the best way to connect both M1 Macs to the Intel NUC via Ethernet? What hardware do I need? A hub? A switch?
Thanks in advance

**Comment by emack999** [comment, marked as answer]
A simple [network switch](https://www.amazon.com/TP-Link-TL-SG105S-Ethernet-Wall-Mount-Protection-dp-B09NMY1RRG/dp/B09NMY1RRG/ref=dp_ob_title_ce) will do.

**Reply by Dimentagon** [reply]
thanks, mate

**Reply by Dimentagon** [reply]
I grabbed a Simple Network Desktop Switch and it works perfectly. Thanks for your answer. 😀
